plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/hms/HiveMetastoreTableOperations.java

```diff
@@ -97,7 +99,17 @@ protected void commitToExistingTable(TableMetadata base, TableMetadata metadata)
             }
         }
         finally {
-            thriftMetastore.releaseTableLock(lockId);
+            try {
+                thriftMetastore.releaseTableLock(lockId);
+            }
+            catch (Exception e) {
+                /* Release lock step has failed. Not throwing this exception, after commit has already succeeded.
+                So, that underlying iceberg API will not do the metadata cleanup, otherwise table will be in unusable state.
+                If configured and supported, the unreleased lock will be automatically released by the metastore after not hearing a heartbeat for a while,
+                or otherwise it might need to be manually deleted from the metastore backend storage.
+                */
+                log.error("Release lock has failed with exception: \"%s\" for table: \"%s\", lockId: \"%s\"", e, tableName, lockId);
```

Comment:
log full stacktrace, not just exception `toString`

also, this may be a bit better wording:

```
log.error(e, "Failed to release lock %s when committing to table %s", lockId, tableName);
```